gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -441,14 +579,23 @@ private BucketListOption(StorageRpc.Option option, Object value) {
       super(option, value);
     }
 
+    /**
+     * Returns an option to specify the maximum number of buckets to be returned.
+     */
     public static BucketListOption maxResults(long maxResults) {
       return new BucketListOption(StorageRpc.Option.MAX_RESULTS, maxResults);
     }
 
+    /**
+     * Returns an option to specify the page token from which to start listing buckets.
+     */
     public static BucketListOption startPageToken(String pageToken) {
       return new BucketListOption(StorageRpc.Option.PAGE_TOKEN, pageToken);
     }
 
+    /**
+     * Returns an option to set a prefix for listed buckets names.
```

Comment:
Returns an option to set a prefix to filter results to buckets whose names begin with this prefix.

Description taken from [here](https://cloud.google.com/storage/docs/json_api/v1/buckets/list).